packages/selenium/src/Resources/allowIframeSafe.js

```diff
@@ -0,0 +1 @@
+axe.configure({ allowedOrigins: ['<same_origin>'] });
```

Comment:
Can this and `allowIframeUnsafe.js` be removed (along with their `.csproj` entries) now that they aren't used?